ParseUI/ParseUI.xcodeproj/project.pbxproj

```diff
@@ -2093,9 +2095,11 @@
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				MARKETING_VERSION = 1.18.0;
 				MTL_ENABLE_DEBUG_INFO = NO;
 				SKIP_INSTALL = YES;
-				TARGETED_DEVICE_FAMILY = "1,2";
+				SUPPORTS_MACCATALYST = NO;
+				TARGETED_DEVICE_FAMILY = 1;
```

Comment:
This suggests we only support iPhone. Should this be 1, 2? (It was changed from 1, 2... why?)

Reply:
Here ParseUI is officially iPhone only. I would think making it available to iPad OS would leave developers under the impression that we've made changes to make ParseUI play nice with some of the iPad OS specific features when in fact we have not. I figured that could be one of our goals for next release is add Apple Sign In and also add iPad OS and Catalyst specific updates.